server/botany/views.py

```diff
@@ -27,14 +27,19 @@
     summary,
 )
 
+def tournament_state():
+    "Flag saying whether tournament is open (True) or closed (False)"
+    close_condition = datetime.now(timezone.utc) < settings.BOTANY_TOURNAMENT_CLOSE_AT
+    start_condition = datetime.now(timezone.utc) > settings.BOTANY_TOURNAMENT_START_AT
+    return close_condition and start_condition
```

Comment:
This now creates `datetime.now()` twice. That could be avoided with a chained comparison:

```
def tournament_is_open():
    """Has the tournament started and has not yet reached the closing date?"""
    start = settings.BOTANY_TOURNAMENT_START_AT
    close = settings.BOTANY_TOURNAMENT_CLOSE_AT
    return start < datetime.now(timezone.utc) < close
```